libraries/vircadia-client/src/error.h

```diff
@@ -0,0 +1,71 @@
+//
+//  error.h
+//  libraries/client/src
+//
+//  Created by Nshan G. on 27 March 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+/// @file
+
+#ifndef LIBRARIES_VIRCADIA_CLIENT_SRC_ERROR_H
+#define LIBRARIES_VIRCADIA_CLIENT_SRC_ERROR_H
+
+#include "common.h"
+
+/// @brief Only one context can exist at a time.
+///
+/// A limitation in the current library implantation, this error code is
+/// returned if user attempts to create the second context.
+///
+/// @return -1
+VIRCADIA_CLIENT_DYN_API int vircadia_error_context_exists();
```

Comment:
There's a question over the use of "_"s in names.

Reply:
Tried to clarified in #1623